### ISSUE TITLE
literal integer are interpreted as int instead of int64.

### ISSUE THREAD

**Issue by ninoles** [hashicorp/consul-template#1143]
### Consul Template version

consul-template v0.19.5 (57b6c71)

### Configuration

```hcl
```

```liquid
{{ scratch.Set "t" 2 }}
{{ range $x := loop (scratch.Get "t") }} -> {{ $x }}
{{ end }}
```

or even:

```liquid
{{ $n := 2 }}
{{ range $x := loop $n }} -> {{ $x }}
{{ end }}
```

### Command

```shell
consul-template -dry -once -template template.txt
```

### Debug output

https://gist.github.com/ninoles/a8c20e7ce709e15e969350d205bd5c57

### Expected behavior

What should have happened?

The template should have run the loop and print `-> 1 -> 2`

### Actual behavior

`.\template.txt: execute: template: :2:33: executing "" at <"t">: wrong type for value; expected int64; got int
`
### Steps to reproduce

1. run the command line as shown above.

**Comment by ninoles** [comment]
I update the bug report with the same example using a simple variable this time... same results.

**Comment by eikenb** [comment]
Thanks for the report @hashicorp/consul-eco 

I'm not sure why the original author changed loop to take int64 instead of int (which it did before that). There is no explanation nor linked issue and it was done 6 months after loop was added (back in 2015). Anyone have a clue?

**Comment by eikenb** [comment]
Fix coming with PR #1255
